app/src/test/drawable/label2d_handler.test.ts

```diff
@@ -484,34 +387,8 @@ test('2d polygons delete vertex and draw bezier curve', () => {
 
   // draw a polygon and delete vertex when drawing
   const canvasSize = new Size2D(1000, 1000)
-  label2dHandler.onMouseMove(new Vector2D(200, 100), canvasSize, -1, 0)
-  label2dHandler.onMouseUp(new Vector2D(200, 100), -1, 0)
-  label2dHandler.onMouseDown(new Vector2D(200, 100), -1, 0)
-  label2dHandler.onKeyDown(new KeyboardEvent('keydown', { key: 'd' }))
```

Comment:
This key event missed.

Reply:
I'll add a commit based on the original logic.

Reply:
@fyu if the modification is fine, let me know to approve it.

Reply:
The change looks great! I hope this new set of API can be used in future tests. Can you check why the tests fail?

Reply:
Yes. I'm looking into it.

Reply:
Fixed

Reply:
Thanks!